src/cdk-experimental/listbox/listbox.ts

```diff
@@ -194,360 +158,382 @@ export class CdkOption<T = unknown> implements ListKeyManagerOption, Highlightab
     'role': 'listbox',
     'class': 'cdk-listbox',
     '[id]': 'id',
-    '(focus)': '_focusActiveOption()',
-    '(keydown)': '_keydown($event)',
-    '[attr.tabindex]': '_tabIndex',
+    '[attr.tabindex]': 'disabled ? null : 0',
```

Comment:
Should the `tabindex` be configurable, but still default to 0?

Reply:
Maybe... It might get a little weird in the roving tabindex case. Would we take the tabindex from the listbox and use it for the active item? What if the options specify their own tabindex different from the listbox?

Reply:
I was thinking of just having `@Input() tabIndex` like the other components so we don't overwrite it if the user set one explicitly. Passing it on through the roving tabindex seems weird to me since users would expect it to be on the listbox. Anyway, I'm also fine with leaving it as is and revisiting it later if we get issue reports.

Reply:
My concern is that at any given time there's only one element that's focusable and represents the listbox as a whole in the tab order (sometimes the listbox, sometimes one of the items). That's going to lead to a sort of wonky experience if the user e.g. binds a special tabindex on the list but not the items. As the focusable element changes the effective tabindex of the listbox will seem to change between whatever the user bound in and the default `0` for the items.

I think inheriting the tabindex from the listbox actually is what users would probably expect, since the listbox and items conceptually represent the same tab stop. What if we do like this:

```
@Directive({host: {'[attr.tabindex]': '_getTabIndex()'}})
class CdkListbox {
  @Input('tabindex') enabledTabIndex = 0;

  _getTabIndex() { ... }
}

@Directive({host: {'[attr.tabindex]': '_getTabIndex()'}})
class CdkOption {
  @Input('tabindex') enabledTabIndex = this.listbox.enabledTabIndex;

  _getTabIndex() { ... }
}
```

This way the default is to copy the tabindex of the listbox, but if for some reason the user wants a different tabindex on the child they can do it

Reply:
@crisbeto does this make sense to you?

Reply:
Yes, I think it's fine as it is. Resolving the conversion.